<launch>
    <rosparam command="load" file="$(find practice_hw)/params/turtle_control.yaml" />
    <node type="turtlesim_node" pkg="turtlesim" name="turtle" output="screen" />
    <node type="turtle_control" pkg="practice_hw" name="node_day3" output="screen" >
        <remap from="/turtle1/cmd_vel_modify" to="/turtle1/cmd_vel"/>
    </node>
</launch>